framework/CodeInterfaces/RELAP5/RELAPparser.py

```diff
@@ -67,6 +67,7 @@ def __init__(self,inputFile,addMinorEdits=False):
     self.deckLines        = {}
     self.maxNumberOfDecks = 0
     self.presentStopTrip = {}
+    self.stopTripNomenclature = '600'
```

Comment:
can you add a comment about this "600" value?

Reply:
added